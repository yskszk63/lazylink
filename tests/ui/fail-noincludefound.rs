#[lazylink::lazylink("c", include="not exists")]
mod libc {}

fn main() {}
