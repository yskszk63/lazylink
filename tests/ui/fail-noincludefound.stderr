error: No such file or directory (os error 2)
 --> $DIR/fail-noincludefound.rs:1:1
  |
1 | #[lazylink::lazylink("c", include="not exists")]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
